src/Symfony/Component/Messenger/Handler/Locator/AbstractHandlerLocator.php

```diff
@@ -21,16 +21,10 @@
 {
     public function resolve($message): callable
     {
-        $messageClass = \get_class($message);
-
-        if (null === $handler = $this->resolveFromClass($messageClass)) {
-            throw new NoHandlerForMessageException(sprintf('No handler for message "%s".', $messageClass));
-        }
-
-        return $handler;
+        return $this->resolveFromClass(\get_class($message));
```

Comment:
Wouldn't it make more sense to just inline `resolveFromClass` in `resolve` at this point?